es-ds-docs/pages/molecules/video.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+const propTableRows = [
+    ['alt-text', 'String', 'n/a', 'Required.'],
+    ['cover-image-url', 'String', 'n/a', 'Required.'],
+    ['embed-url', 'String', 'n/a', 'Required.'],
+];
+
+const { $prism } = useNuxtApp();
+const compCode = ref('');
+const docCode = ref('');
+onMounted(async () => {
+    if ($prism) {
+        const compSource = await import('@energysage/es-ds-components/components/es-video.vue?raw');
+        // eslint-disable-next-line import/no-self-import
+        const docSource = await import('./video.vue?raw');
+
+        compCode.value = $prism.normalizeCode(compSource.default);
+        docCode.value = $prism.normalizeCode(docSource.default);
+        $prism.highlight();
+    }
+});
+</script>
+
+<template>
+    <div>
+        <h1>Video</h1>
+        <div class="my-500">
+            <b-row>
+                <!-- TODO: set explicit width & height on on es-video container to avoid CLS -->
+                <b-col md="8">
+                    <es-video
+                        :alt-text="`
+                            ME from EnergySage talking to you about how to decide if solar panels
+                            are worth it for your needs
+                        `"
+                        cover-image-url="https://a-us.storyblok.com/f/1006159/1280x720/2cb0626421/are-solar-panels-worth-it_.jpg/m/810x0/filters:format(jpg):quality(70)"
+                        embed-url="https://www.youtube.com/embed/srBDwFZqOi0" />
```

Comment:
Do we want to use real ES videos? Could stick with the cat theme here

Reply:
Yea that's why I ended up using https://loremflickr.com/200/200?random=3 🙁

Reply:
IMO we should use real-world EnergySage use cases whenever possible to quickly make sense to our internal users of the design system, without explanation, what the purpose of a component is - and when doing so wouldn't compromise any internal information, and when such an example is already public, as our YouTube videos are.

Reply:
Could be something to discuss in the design guild, I had thought we were moving away from real ES examples but could be wrong

Reply:
@hroth1994 Totally agree. In the interim though, I reverted back to what was there before as the ticket was to upgrade the component for vue3.